tasks/configure.js

```diff
@@ -1,33 +1,37 @@
 'use strict';
 
-let packageJson = require(process.cwd() + '/package.json');
-let findService = require('../lib/find-service');
-let herokuAuthToken = require('../lib/heroku-auth-token');
-let normalizeName = require('../lib/normalize-name');
-let vault = require('../lib/vault');
-let fetchres = require('fetchres');
+const packageJson = require(process.cwd() + '/package.json');
+const findService = require('../lib/find-service');
+const herokuAuthToken = require('../lib/heroku-auth-token');
+const normalizeName = require('../lib/normalize-name');
+const vault = require('../lib/vault');
+const pipelines = require('../lib/pipelines');
+const HerokuConfigVars = require('../lib/heroku-config-vars');
 
 const FORBIDDEN_ATTACHMENT_VARIABLES = [
 	'DATABASE_URL'
 ];
 
 const DEFAULT_REGISTRY_URI = 'https://next-registry.ft.com/v2/';
```

Comment:
(separate to this PR)

Should we remove the ability to change the registry URL? It's legacy from the last time we migrated registries. AFAIK everything can/should just use the v2 one now - not sure whether it's ever going to be useful again later.

Thoughts @sjparkinson ?

Reply:
I believe this is so that Enterprise Tools can access their respective registries?